[10.0] l10n_ch_hr_payroll: update some default values

asaunier opened the pull request:
Fixes https://github.com/OCA/l10n-switzerland/issues/390

I have only changed OBP minimum legal (``lpp_min``) and UI Maximum limit (``ac_limit``).
OAI/II/IC rate was already at 5.125%

The second commit edc7d6e is not really mandatory: tests pass without it. But it makes sense to update the example data with the new ``ac_limit`` and ``lpp_min`` default values.
But then the ``wage_fulltime`` of the example is no longer above the UI maximum limit => we had to upgrade this wage and the related tested values.

asaunier committed:
[10.0] l10n_ch_hr_payroll: update ac_limit and lpp_min default values

mpanarin reviewed (approved):
other than, imo, not the best commit message - LGTM
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#commit-message

asaunier commented:
@yvaucher @mpanarin Thanks for reviewing!

I have squashed the 2 original commits into a single one to make the PR simpler. I had initially made 2 distinct commits because the changes in ``test_max_lpp.py`` were quite cumbersome because of many hard-coded tested values I had to update.

I have also changed the PR and commit messages in order to follow the guidelines, even though the commit message is a bit longer than 50 chars but it seems reasonable to me.

asaunier commented:
The suggested changes cannot be cherry-picked to Odoo 9 because the addon has quite changed from 9 to 10. Here are proposed changes for version 9: https://github.com/OCA/l10n-switzerland/pull/395
